backend/note_taking/views/note_views.py

```
@@ -53,6 +53,13 @@ def perform_update(self, serializer):
         
         serializer.save()
 
+class RetrieveNoteEndpoint(RetrieveAPIView):
+    serializer_class = NoteSerializer
+    permission_classes = [IsAuthenticated]
+
+    def get_queryset(self):
+        queryset = Note.objects.filter(notebook__owner=self.request.user)
```

Ruff (0.12.2)

Local variable `queryset` is assigned to but never used

Remove assignment to unused variable `queryset`

(F841)